app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def sign_in(user)
+    remember_token = User.new_token
+    cookies.permanent[:remember_token] = remember_token
+    user.update_attribute(:remember_digest, remember_token)
+  end
```

Comment:
Team if you compare what you are doing here with the `create_remember_token` method in the User model they are different but they should be the same, let me explain this: when you execute the `sing_in` method you are just assigning to the `remember_digest` attribute the value generated by `SecureRandom.urlsafe_base64` but in your `create_remember_token` method you are assigning the `Digest::SHA1.hexdigest(remember_token)` and that is the right one, hence my suggestion is to call in the `sign_in` helper the `create_remember_token` method, bear in mind that the method should be static to achieve this.